test-cases/gemini/gemini-basic-3h.yaml

```diff
@@ -1,20 +1,20 @@
 test_duration: 500
 n_db_nodes: 1
 n_loaders: 1
-n_monitor_nodes: 1
+n_monitor_nodes: 0
 instance_type_db: 'i3.large'
 
 user_prefix: 'gemini-basic-3h'
-failure_post_behavior: keep
+failure_post_behavior: destroy
 store_results_in_elasticsearch: False
 # gemini
 # cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
 # the below cmd runs about 3 hours
-gemini_cmd: "gemini -d -n 500000 -c 20 -p 100 -m mixed -f"
-gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-latest'
+gemini_cmd: "gemini -d --duration 10800s -c 20 -p 100 -m mixed -f"
+gemini_url: 'http://downloads.scylladb.com.s3.amazonaws.com/gemini/0.9.2/gemini_0.9.2_Linux_x86_64.tar.gz'
 gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
 
 db_type: mixed_scylla
-instance_type_db_oracle: 'i3.8xlarge'
-ami_id_db_oracle: 'ami-0081311880c437b65' # ScyllaDB 2.3.3 us-east-1 AMI
+instance_type_db_oracle: 'i3.large'
+# ami_id_db_oracle: 'ami-0081311880c437b65' # ScyllaDB 2.3.3 us-east-1 AMI
```

Comment:
Why did you comment it?
What do we use for the test-oracle?

Reply:
This parameter is configured in Job configuration, and could be set upon run. In job i use the the 2.3.5 version of Scylla.

Reply:
Even if this is the case, this is not the way to do it.
You could replacve the value with AMI_ID_TEST_ORACLE

However, I don't think this is something we want to replace every run or allow people to change it often so it won't cause confusions.
If you want to update to 2.3.5 just update here.

Reply:
done